library/src/main/android/permissions/dispatcher/PermissionUtils.java

```diff
@@ -96,7 +96,7 @@ public static boolean hasSelfPermissions(Context context, String... permissions)
      * @see #hasSelfPermissions(Context, String...)
      */
     private static boolean hasSelfPermission(Context context, String permission) {
-        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.M && Build.MANUFACTURER.equalsIgnoreCase("Xiaomi")) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.M && "Xiaomi".equalsIgnoreCase(Build.MANUFACTURER)) {
```

Comment:
sorry,the `Build.VERSION.SDK_INT > Build.VERSION_CODES.M` should be `Build.VERSION.SDK_INT >= Build.VERSION_CODES.M`;It's should be work in Build.VERSION_CODES.M and above